django/contrib/admin/templatetags/admin_urls.py

```diff
@@ -1,3 +1,4 @@
+import urllib
```

Comment:
Could quote and unquote not be added to the `from urllib.parse import ...` line below, to avoid this import for the same namespace?

Reply:
Thank you for review. I fix the import in the next commit.